# MISC
CORRECTION_SIZE: 4

# FILTER
MAP_SIZE: 500
PIX_NOISE: 1.0
PIX_NOISE_SIMUFACTOR:  0.0

# LANDMARKS
D_MIN: .6
REPARAM_TH: 0.1 

GRID_HCELLS: 6
GRID_VCELLS: 4
GRID_MARGIN: 5
GRID_SEPAR: 7

RELEVANCE_TH: 2.0
MAHALANOBIS_TH: 3.0
N_UPDATES_TOTAL: 25
N_UPDATES_RANSAC: 22
#N_UPDATES_TOTAL: 17
#N_UPDATES_RANSAC: 15
N_INIT: 10
N_RECOMP_GAINS: 2
RANSAC_LOW_INNOV: 1.0 

RANSAC_NTRIES: 6
MULTIPLE_DEPTH_HYPOS: 1

# RAW PROCESSING
HARRIS_CONV_SIZE: 5
HARRIS_TH: 5.0
HARRIS_EDDGE: 1.4

MULTIVIEW_DESCRIPTOR: 0
DESC_SIZE: 21
DESC_SCALE_STEP: 1.5
DESC_ANGLE_STEP: 5
DESC_PREDICTION_TYPE: 1

PATCH_SIZE: 13
MAX_SEARCH_SIZE: 10000
KILL_SEARCH_SIZE: 100000
MATCH_TH: 0.70
MIN_SCORE: 0.65
HI_MATCH_TH: 0.90
HI_LIMIT: 100
PARTIAL_POSITION: 0.25
